plugin/type_uuid/mysql-test/type_uuid/func_sys_guidv7_plugin.test

```diff
@@ -0,0 +1,30 @@
+--echo #
+--echo # Start of 10.5 tests
```

Comment:
a) 11.5
b) the convention to use "Start of .. tests" is limited. End of tests (with right version) is the main factor for easy of merging.